client/src/modules/patients/edit/edit.html

```diff
@@ -396,6 +396,26 @@ <h4><span translate>FORM.LABELS.CURRENT_LOCATION</span></h4>
                   </div>
                 </div>
 
+                <div class="form-group" ng-class="{'has-error' : patientDetails.employer.$invalid && PatientRegistrationForm.$submitted}">
```

Comment:
What is `patientDetails.employer`? 

It should say `patientDetails.health_zone`.  The `has-error` class should match the `name` of the `<input>` that it wraps.

Reply:
This still needs to addressed, afterwards we can merge it.